src/utils/search.js

```diff
@@ -30,22 +30,21 @@ export let transformNodeFromField = args => node => ({
   ...newNodeFromField(args),
 })
 
-export let indent = (Tree, parent, node, skipDefaultNode) => {
+export let indent = async (Tree, parent, node, skipDefaultNode) => {
   // Reactors:
   //   OR -> And, nothing
   //   AND -> OR, others if has value
   //   to/from NOT, others if has value
   let join = (parent || node).join
-  let key = node.key
   let path = _.toArray(node.path)
-
-  Tree.mutate(path, { key: `${key}-${join}-group` })
```

Comment:
this is a good catch - I don't think we've ever had a case to `mutate` a `key`, and I'm honestly not sure if it worked 🤔